libs/pascal_voc_io.py

```diff
@@ -152,7 +156,6 @@ def parse_xml(self):
         assert self.file_path.endswith(XML_EXT), "Unsupported file format"
         parser = etree.XMLParser(encoding=ENCODE_METHOD)
         xml_tree = ElementTree.parse(self.file_path, parser=parser).getroot()
-        filename = xml_tree.find('filename').text
```

Comment:
Rationale: Variable is never used. Not sure whether this is a code artifact or points to a bug. May make sense to look into it

**To do:**

- [ ] check if this is correct behavior